main.go

```diff
@@ -0,0 +1,249 @@
+package main
```

Comment:
def gotta get rid of this file before moving on. I'd like to see a switch back to `jsonschema` before doing a final pass